### ISSUE TITLE
Add link and referr to page of update fw and main page for changelog into gui Web

### ISSUE THREAD

**Issue by lostboy86** [ktomy/nightscout-clock#10]
Add link and refer to page of update fw and main page for changelog into gui Web of device, for update on the same page or passtrough quickly to the correct link / page.

**Comment by ktomy** [comment]
Yes, will do. "fix the docs" is what I will do after I will finish the alarms.
The priority now is:
- fix the ImprovWifi (no need of the Ap mode to set up the clock)
- test and production ze alarms
- improve the docs
- create good tutorial /how-to video

**Comment by lostboy86** [comment]
Maybe i write not good,

i mean :  place a button for update fw into the Gui web of ulanzi, for
upgrade fw quickly.
If over the mouse of button, see a simple changelog., same of github
section for every commits.


Il giorno mar 7 mag 2024 alle ore 22:15 Alin Artiom Kenibasov <
***@***.***> ha scritto:
